ledger/test-common/src/main/daml/semantic/Limits.daml

```diff
@@ -0,0 +1,43 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Limits where
+
+import DA.Map qualified as M
+
+template WithMap
+  with
+    owner : Party
+    items : M.Map Text Party
+  where
+    signatory owner
+    nonconsuming choice WithMap_Noop : ()
```

Comment:
what's the purpose of this no-op nonconsuming choice?

Reply:
if it's for forcing deserialization in during interpretation, isn't it enough to just archive it?

Reply:
Well, when it is a noop you can run it a 1000 times in parallel and that's what you can do in dev experiments based on that contarct